src/assets/i18n/nl.json

```diff
@@ -193,6 +195,8 @@
   "LABEL_SEARCH_RESULTS_EDIT_FORM_DETAIL_PAGE_IMAGE_WIDTH": "Breedte",
   "LABEL_SEARCH_RESULTS_EDIT_FORM_DETAIL_PAGE_IMAGE_HEIGHT": "Hoogte",
   "LABEL_SEARCH_RESULTS_EDIT_FORM_DETAIL_PAGE_IMAGE_DEFAULT_IMAGE": "Altijd een afbeelding tonen",
+  "LABEL_SEARCH_RESULTS_EDIT_FORM_DETAIL_PAGE_IMAGE_UIT_LOGO": "UiT-Logo",
```

Comment:
Here it's capitalized `Logo`, but on line 154 it's lowercase `logo`. Should this be more consistent?